packages/notifier/src/notifier.js

```diff
@@ -163,54 +127,55 @@ export const makeNotifierFromAsyncIterable = asyncIterableP => {
 
   /** @type {Promise<UpdateRecord<T>>|undefined} */
   let optNextPromise;
-  /** @type {UpdateCount} */
-  let currentUpdateCount = 1; // avoid falsy numbers
-  /** @type {UpdateRecord<T>|undefined} */
+  /** @type {UpdateCount & bigint} */
+  let currentUpdateCount = 0n;
```

Comment:
What’s the rationale for 0n vs 1n?

Reply:
It's mostly for better alignment with publish kit, in which `publishCount` actually counts the number of publishes.

The number shouldn't really matter, but `0n` is more... natural. :sunglasses: